client/state/data-layer/wpcom/read/streams/test/index.js

```diff
@@ -17,6 +17,7 @@ jest.mock( '@wordpress/warning', () => () => {} );
 
 describe( 'streams', () => {
 	const action = deepfreeze( requestPageAction( { streamKey: 'following', page: 2 } ) );
+	const ISO_DATE_MATCHER = expect.stringMatching( /^\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}Z$/ );
```

Comment:
This is a regex check to see if we've successfully matched a date in the format: "2023-11-02T00:00:00Z"